doc/Requirements Specification/Parts/Terms and Definitions.lyx

```diff
@@ -1204,6 +1249,50 @@ newglossaryentry{Kieker}{
 }
 \end_layout
 
+\begin_layout Plain Layout
+
+\end_layout
+
+\begin_layout Plain Layout
+
+
+\backslash
+newglossaryentry{measurement software}{
```

Comment:
It’s interesting acutally. Because “measuring software” is referenced by the “Kieker” glossary entry. But obiously, this does not leasd “measuring software” to be included.

This will, however not be a problem at this point, as #73 will use “measuring software”. But we should keep this in mind.